src/yz_kv.erl

```diff
@@ -460,3 +463,131 @@ is_owner_or_future_owner(P, Node, Ring) ->
 -spec is_service_up(atom(), node()) -> boolean().
 is_service_up(Service, Node) ->
     lists:member(Service, riak_core_node_watcher:services(Node)).
+
+%% @private
+%%
+%% @doc Check if object has 2.0 CRDT datatype entry or property for
+%%      strong consistency.
+-spec is_datatype_or_consistent(obj()) -> boolean().
+is_datatype_or_consistent(Obj) ->
+    case riak_kv_crdt:value(Obj) of
```

Comment:
Similar to the `is_strongly_consistent/1` below, you could check the bucket properties only to determine if it's a CRDT. Calling `riak_kv_crdt:value/1` will perform a deserialization of the value, which is extra work if you are not intending to use the result.